web/models.py

```
@@ -2662,6 +2662,138 @@
+        User, on_delete=models.SET_NULL, null=True, blank=True, related_name="reviewed_submissions"
+    )
+
+    class Meta:
+        ordering = ["-submitted_at"]
+        verbose_name = "Work Submission"
+        verbose_name_plural = "Work Submissions"
+
+    def __str__(self) -> str:
+        return f"{self.student.username}'s {self.work_type.name}: {self.title}"
+
+    def save(self, *args, **kwargs) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)